package.json

```
@@ -17,14 +17,9 @@
     "start": "node tasks/serve.js",
     "test": "npm run lint && jest",
     "lint": "npm run lint:js && npm run lint:scss",
-    "lint:js": "standard",
+    "lint:js": "eslint --cache --cache-location .cache/eslint --cache-strategy content --color --ignore-path .gitignore -- \"**/*.js\"",
```

Seems to work with the `--` but probably take it out?

```suggestion
    "lint:js": "eslint --cache --cache-location .cache/eslint --cache-strategy content --color --ignore-path .gitignore \"**/*.js\"",
```

---

Nice spot, done!